src/vs/workbench/browser/parts/compositeBar.ts

```diff
@@ -414,23 +414,17 @@ export class CompositeBar extends Widget implements ICompositeBar {
 	}
 
 	areBadgesEnabled(compositeId: string): boolean {
```

Comment:
is this function used anywhere else? I only see it below and its a one liner now

Reply:
Yes it is used in some other places in compositeBar and compositeBarAction. It's a one liner, but I feel like it's a more concise function to call. Let me know if you just want me to paste the one liner everywhere. This also makes refactoring easier